images/site-packages/aiomqtt/x1plus_example.py

```diff
@@ -0,0 +1,73 @@
+import x1plus.utils
+import asyncio
+import aiomqtt 
+import json
+import ssl
+import os
+
+#usage of aiomqtt outside of x1plusd - example
+#pub(): publishes gcode_line command that toggles fan speed
+#sub(): subscribes to wildcard topic /device/#
+
+def write_flag(val):
+	with open('/tmp/gcode_flag', 'w') as file:
+		file.write(val)
+
+def check_flag():
+    if os.path.exists('/tmp/gcode_flag'):
+        with open('/tmp/gcode_flag', 'r') as file:
+            flag = file.read()
+            return flag
+    return None
+
+def make_payload():
+    speed = check_flag()
+    speed = '255' if speed == '0' else '0' if speed == '255' else '0'
+    write_flag(speed)
+    return json.dumps({
+        "print": {
+            "command": "gcode_line",
+            "sequence_id": 0,
+            "param": f"M106 P1 S{speed}"
+        }
+    })
+
+async def sub():
+    try:
+        async with aiomqtt.Client(
+            '127.0.0.1', 
+            8883,
+            username='bblp', 
+            password=x1plus.utils.access_code(), 
+            tls_context=ssl.create_default_context(cafile='/usr/etc/system/certs/ssl-ca-bbl.pem'), 
+            tls_insecure=True
+        ) as client:
+            await client.subscribe("device/#")
+            
+            async for message in client.messages:
+                topic = message.topic.value
+                msg = message.payload.decode("utf-8")
+                print(f"{topic}\n{msg}")
+            
+    except Exception as e:
+        print(f"Error subscribing: {e}")
```

Comment:
this definitely should not live in site-packages!  site_packages is things we have vendored in and should only EXACTLY be external code!

Reply:
yea this definitely was not supposed to be there! I do have an actual example that's useful, but it doesn't belong here